client/blocks/get-apps/desktop-download-card.jsx

```diff
@@ -122,15 +130,17 @@ class DesktopDownloadCard extends Component {
 					'Also available for: ' +
 						'{{firstAvailableLink}}Linux (.deb){{/firstAvailableLink}}, ' +
 						'{{secondAvailableLink}}Windows{{/secondAvailableLink}}, ' +
-						'{{thirdAvailableLink}}Mac{{/thirdAvailableLink}}.',
+						'{{thirdAvailableLink}}Mac (Intel){{/thirdAvailableLink}}.' +
+						'{{fourthAvailableLink}}Mac (Apple Silicon){{/fourthAvailableLink}}.',
 					{ components: this.getTranslateComponents( platform ) }
 				);
 			default:
 				return translate(
 					'Also available for: ' +
-						'{{firstAvailableLink}}MacOS{{/firstAvailableLink}}, ' +
-						'{{secondAvailableLink}}Linux (.tar.gz){{/secondAvailableLink}}, ' +
-						'{{thirdAvailableLink}}Linux (.deb){{/thirdAvailableLink}}.',
+						'{{firstAvailableLink}}MacOS (Intel){{/firstAvailableLink}}, ' +
+						'{{secondAvailableLink}}MacOS (Apple Silicon){{/secondAvailableLink}}, ' +
+						'{{thirdAvailableLink}}Linux (.tar.gz){{/thirdAvailableLink}}, ' +
+						'{{fourthAvailableLink}}Linux (.deb){{/fourthAvailableLink}}.',
 					{ components: this.getTranslateComponents( platform ) }
 				);
```

Comment:
Well done following the same pattern we were using.

We could consider translating only 'Also available for: ' and returning the rest without any translation.

<img width="507" alt="Screenshot 2023-03-14 at 10 53 44" src="https://user-images.githubusercontent.com/779993/224984424-5df57614-a441-4677-80ef-7eeb892a1535.png">